.github/workflows/publish-draft-release.yml

```diff
@@ -65,49 +54,52 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: ${{ matrix.runtime }}-runtime
-          path: "${{ steps.build-runtime.outputs.wasm }}"
+          path: |
+            ${{ steps.srtool_build.outputs.wasm }}
+            ${{ steps.srtool_build.outputs.wasm_compressed }}
 
   publish-draft-release:
     runs-on: ubuntu-latest
-    needs: ['get-rust-versions', 'build-runtimes']
+    needs: ["get-rust-versions", "build-runtimes"]
     outputs:
       release_url: ${{ steps.create-release.outputs.html_url }}
       asset_upload_url: ${{ steps.create-release.outputs.upload_url }}
     steps:
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-        path: polkadot
-    - name: Set up Ruby 2.7
-      uses: actions/setup-ruby@v1
-      with:
-        ruby-version: 2.7
-    - name: Download srtool json output
-      uses: actions/download-artifact@v2
-    - name: Generate release text
-      env:
-        RUSTC_STABLE: ${{ needs.get-rust-versions.outputs.rustc-stable }}
-        RUSTC_NIGHTLY: ${{ needs.get-rust-versions.outputs.rustc-nightly }}
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      run: |
-        gem install changelogerator git toml
-        ruby $GITHUB_WORKSPACE/polkadot/scripts/github/generate_release_text.rb | tee release_text.md
-    - name: Create draft release
-      id: create-release
-      uses: actions/create-release@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        tag_name: ${{ github.ref }}
-        release_name: Polkadot ${{ github.ref }}
-        body_path: ./release_text.md
-        draft: true
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          path: polkadot
+      - name: Set up Ruby 2.7
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - name: Download srtool json output
+        uses: actions/download-artifact@v2
+      - name: Generate release text
+        env:
+          RUSTC_STABLE: ${{ needs.get-rust-versions.outputs.rustc-stable }}
+          RUSTC_NIGHTLY: ${{ needs.get-rust-versions.outputs.rustc-nightly }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gem install changelogerator git toml
+          ruby $GITHUB_WORKSPACE/polkadot/scripts/github/generate_release_text.rb | tee release_text.md
```

Comment:
this looks huge

Reply:
The PR did not change that part. It looks like the identation changed though.
I did run some formatting.

Reply:
Then PR is good, but in general, it's something that would need some love.